reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md

```diff
@@ -148,25 +176,32 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.String
+
 You can pipe a string to this cmdlet.
 
 ## OUTPUTS
 
 ### Microsoft.PowerShell.Commands.ByteCollection
-This cmdlet returns a **ByteCollection**.
-This object represents a collection of bytes.
-It includes methods that convert the collection of bytes to a string formatted like each line of output returned by **Format-Hex**.
-If you specify the *Path* or *LiteralPath* parameter, the object also contains the path of the file that contains each byte.
+
+This cmdlet returns a **ByteCollection**. This object represents a collection of bytes. It includes
+methods that convert the collection of bytes to a string formatted like each line of output returned
+by **Format-Hex**. If you specify the *Path* or *LiteralPath* parameter, the object also contains
```

Comment:
`*Path* or *LiteralPath* parameter` - should be bold for parameters

Reply:
Thanks. Updated the formatting and submitted the changes for review.